app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -769,6 +811,19 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         hideReverseGeolocateResult()
     }
 
+    override fun overridePlaceFeaturePosition(feature: Feature) {
+        if (poiTapPoint != null) {
+            val geometry = Geometry()
+            val coordinates = ArrayList<Double>()
+            var coords = mapController?.coordinatesAtScreenPosition(
```

Comment:
Again, maybe there is a way to rewrite this section with less `!!`. Its not super critical since we control the Tangram library and trust it. But if Tangram was a third party product I would never want to use `!!` on nullable return values ;)